tests/album-create.test.js

```diff
@@ -0,0 +1,49 @@
+// tests/album-create.js
+const { expect } = require("chai");
+const request = require("supertest");
+const getDb = require("../src/services/db");
+const app = require("../src/app");
+
+describe("create album", () => {
+  let db;
+  let artist;
+  beforeEach(async () => (db = await getDb()));
+
+  beforeEach(async () => {
+    db = await getDb();
+    await db.query("INSERT INTO Artist (name, genre) VALUES(?, ?)", [
+      "Tame Impala",
+      "rock",
+    ]);
+
+    [artist] = await db.query("SELECT * FROM Artist");
+  });
+
+  afterEach(async () => {
+    await db.query("DELETE FROM Album");
+    await db.query("DELETE FROM Artist");
+    await db.close();
+  });
+
+  describe("/album", () => {
+    describe("POST", () => {
+      it("creates a new album in the database", async () => {
```

Comment:
Could there be any other tests included here? For example what would happen if an album was attempted to be added to an artist that isnt in the database? Or if the name or year fields were blank?